cmd/completion.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"errors"
+	"os"
+)
+
+// bashCompletionCmd represents the completion command
+var completionCmd = &cobra.Command{
+	Use:       "completion [shell]",
+	Short:     "Output shell completion code for the specified shell (bash or zsh)",
+	Long:      "Output shell completion code for the specified shell (bash or zsh)",
+	ValidArgs: []string{"bash"},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return cmd.Usage()
+		}
```

Comment:
Use `cobra.Args: cobra.ExactArgs(1),` in the value of `completionCmd` instead of checking for the length of `args` here.